src/Props/schedule/schedule.ts

```diff
@@ -12,7 +12,9 @@ export const schedule: ScheduleDate[] = [
   {
     date: "Friday, January 14, 2022",
     events: [
+      { name: "Swag Distribution Starts", time: "" },
       { name: "Event Begins", time: "5:00 PM" },
```

Comment:
Last thing, let's switch the times here so the 5PM is at the top.

Reply:
the jira ticket says the start time is at 3:00PM are we sure we want to remove the time / move it and imply it starts after 5?

Reply:
Oh my bad, then this should have the time at 3:00PM for Swag Distribution and remain the same. I must have gotten confused initially with the 2 other 5:00 times while reviewing this. My mistake, sorry!

Reply:
np! just making sure